src/python/ensembl/io/genomio/annotation/update_description.py

```diff
@@ -103,15 +101,16 @@ def load_descriptions(
         func_file: JSON file with the annotation information.
         report: Print the mapping of changes to perform in the standard output?
         do_update: Update core database?
+        match_xrefs: If the IDs do not match, try to match an Xref ID instead?
```

Comment:
report and match_xrefs end with a '?'. Doesn't really make sense or read well. Couldn't they be removed and or reworded